README.md

```diff
@@ -22,27 +22,27 @@ The official MySQL and MariaDB use following environment variables to define the
 * `MYSQL_ROOT_PASSWORD` - This variable is mandatory and specifies the password that will be set for the `root` superuser account.
 * `MYSQL_USER`, `MYSQL_PASSWORD` - These variables are optional, used in conjunction to create a new user and to set that user's password.
 
-## Docker hub tags
+## Supported Docker hub tags
 
-You can use following tags on Docker hub:
+The following tags are available:
 
-* `latest` - latest stable release
-* `4.8` - latest stable release for the 4.8 version
-* `4.8.5` - specific patch release for the 4.8 version
+* `latest`, `fpm`, and `fpm-alpine` are always the most recent released version
+* Major versions, such as `5`, `5-fpm`, and `5-fpm-alpine`
+* Specific minor versions, such as `5.0`, `5.0-fpm`, and `5-fpm-alpine`
+* Specific patch versions, such as `5.0.0`, `5.0.0-fpm`, and `5.0.0-alpine`. Note that, on rare occasion,
+there may be an intermediary "docker-only" release, such as 4.9.2-1
```

Comment:
IMHO thy are superfluous because you can also [pull by digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier).

Reply:
Hi @J0WI, thanks for the feedback. Sorry I'm a little unclear here, which parts would you say are superfluous — all of these (5, 5.0, and 5.0.0), or only the last line where I refer to 4.9.2-1? I am familiar with pulling by digest, but I'm not sure what form it would take for us to communicate that to users. How would a user know to pull a particular digest over a specific tag they desire? Can you help clear this up for me a bit?

Thank you

Reply:
As a docker user I would never pull by digest because it is complicated for a simple result
So tags are a must have for me 

I like how devilbox manages tags for different variants, making an upgrade very simple
https://hub.docker.com/r/devilbox/php-fpm/tags

Reply:
> which parts would you say are superfluous — all of these (5, 5.0, and 5.0.0), or only the last line where I refer to 4.9.2-1?

Only for `4.9.2-1`.

Reply:
Describing that this type of release exists is superfluous or the release itself is superfluous?

Reply:
IMHO the release itself is superfluous.

Reply:
The alternative that I see would be to force-push a new tag and I'm not sure how Docker deals with a moving Git tag. I've wondered about this before but I'm not sure what the "correct" solution is.

Reply:
can the regex have a non capturing group so we can have x.y.z-N and only use x.y.z ?